cashapppay/src/main/java/com/adyen/checkout/cashapppay/internal/ui/DefaultCashAppPayDelegate.kt

```diff
@@ -224,7 +224,7 @@ constructor(
         val amount = componentParams.amount
 
         // We don't create an OneTimeAction for transactions with no amount
-        if (amount.value <= 0) return null
+        if (amount?.value == null || amount.value <= 0) return null
```

Comment:
I guess `amount.value <= 0` should nott apply anymore after we removed `EMPTY` right?

Reply:
Makes sense